starter-code/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Will's Cool Simpsons Page",
```

Comment:
This is not matching a valid name cause it has spaces, so wasn't able to do ```npm install```. Better use kebab-case